specification/cognitiveservices/data-plane/TranslatorText/Info.md

```diff
@@ -0,0 +1,4 @@
+# Microsoft Translator API V3
```

Comment:
Any specific reason we need this file?
This is specifications repo, this kind of announcement should be in documents (or swaggers "info"."description" field).